lib/fxpay/errors.js

```diff
@@ -0,0 +1,75 @@
+(function() {
+  'use strict';
+
+  var exports = fxpay.errors = {};
+
+  // All error classes will implicitly inherit from this.
+  exports.FxPayError = createError('FxPayError');
+
+
+  exports.AddReceiptError = createError('AddReceiptError');
+  exports.AppOriginMissing = createError('AppOriginMissing');
+  exports.AppReceiptMissing = createError('AppReceiptMissing');
+  exports.FailedWindowMessage = createError('FailedWindowMessage');
+  exports.IncorrectUsage = createError('IncorrectUsage');
+  exports.InvalidApp = createError('InvalidApp');
+  exports.InvalidJWTData = createError('InvalidJWTData');
+  exports.InvalidPayProviderURL = createError('InvalidPayProviderURL');
```

Comment:
Minor nit -  elsewhere I think we've got `fooUrl` not `fooURL`. Not sure if URL is better but we probably should look to be consistent.